src/vs/workbench/contrib/chat/browser/attachments/promptInstructions/promptInstructionsCollectionWidget.ts

```diff
@@ -35,10 +35,8 @@ export class PromptInstructionsAttachmentsCollectionWidget extends Disposable {
 	 * Subscribe to the `onAttachmentsChange` event.
 	 * @param callback Function to invoke when number of attachments change.
 	 */
-	public onAttachmentsChange(callback: () => unknown): this {
-		this._register(this._onAttachmentsChange.event(callback));
-
-		return this;
+	public onAttachmentsChange(callback: () => unknown): IDisposable {
```

Comment:
Why have this method instead of exposing the event? That's how we do it in most places

Reply:
A left-over, will fix in an upcoming PR 👍